### PR TITLE
feat/Network chip

### PR DESCRIPTION
**Pull request by rinzler99**
Task link: https://toruslabs.atlassian.net/browse/PD-1323

Demo: https://www.loom.com/share/86b06f57a9ad4f54a59b7b46eadd7001

### CODE REVIEW

File: src/components/helpers/NetworkDisplay/NetworkDisplay.vue [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
+        <div class="d-flex network-chip align-center" :class="[chipClass, minimal ? 'network-chip--minimal' : '']" v-on="on">
+          <v-icon v-if="showIcon" left>$vuetify.icons.network</v-icon>
+          <span class="network-chip__name text-clamp-one" :class="{ 'network-chip__name--mobile': $vuetify.breakpoint.xsOnly }">
+            {{ $vuetify.breakpoint.xsOnly && !minimal ? shortSelectedNetwork : selectedNetwork }}
+          </span>
+          <v-icon right>$vuetify.icons.selectNew</v-icon>
+        </div>
+      </template>
+      <v-list class="select-item-list overflow-y-auto" style="max-height: 240px">
+        <v-list-item v-for="host in supportedNetworks" :key="host.networkName" class="select-coin-eth" @click="changeNetwork(host)">
```
Comment (lionellbriones) [inline src/components/helpers/NetworkDisplay/NetworkDisplay.vue:20]: use a different variable name for host

File: src/containers/WalletTransfer/WalletTransfer.vue [hunk 2]
```diff
@@ -1337,6 +1337,7 @@ export default {
       this.changeSelectedToCurrency(0)
     },
     async sendCoin() {
+      debugger
```
Comment (lionellbriones) [inline src/containers/WalletTransfer/WalletTransfer.vue:1340]: remove debugger

File: src/components/helpers/NetworkDisplay/NetworkDisplay.vue [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
-  <div class="d-flex network-chip align-center" :class="[chipClass, minimal ? 'network-chip--minimal' : '']">
-    <v-icon v-if="showIcon">$vuetify.icons.network</v-icon>
+  <div v-if="!isNetworkPill" class="d-flex network-chip align-center" :class="[chipClass, minimal ? 'network-chip--minimal' : '']">
+    <v-icon v-if="showIcon" left>$vuetify.icons.network</v-icon>
     <span class="network-chip__name text-clamp-one" :class="{ 'network-chip__name--mobile': $vuetify.breakpoint.xsOnly }">
       {{ $vuetify.breakpoint.xsOnly && !minimal ? shortSelectedNetwork : selectedNetwork }}
     </span>
   </div>
+  <div v-else>
+    <v-menu transition="slide-y-transition" class="select-menu" offset-y rounded="lg">
```
Comment (lionellbriones) [inline src/components/helpers/NetworkDisplay/NetworkDisplay.vue:9]: use pointer as cursor so user will know it's clickable on hover

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**